### DISCUSSION TITLE
Stack Data Type

### DISCUSSION THREAD

**Discussion by mattkeyeswc** [microsoft/ALAppExtensions#19186] [Ideas] [answered]
Thank you developers for the new(ish) List, Dictionary, etc., data types.  Would it be possible to have a Stack data type in the future?

**Comment by bjarkihall** [comment]
I think stack and queue have been discussed before and I also think I recall it being in the BC ideas waiting for more upvotes. Another possibility is using a homemade codeunit as an abstraction - or even create a System App module which wraps the standard dotnet types, so everyone don't have to create their own implementations.

**Comment by JesperSchulz** [comment, marked as answer]
That question is not for this repository, I'm afraid. As Bjarki states, that would be a case of [BCIdeas ](aka.ms/bcideas), or maybe the [AL repository](https://github.com/microsoft/AL/).
